[esx]/es_extended/server/classes/player.lua

```diff
@@ -19,6 +19,12 @@ function CreateExtendedPlayer(playerId, identifier, group, accounts, inventory,
 	if Config.Multichar then self.license = 'license'.. identifier:sub(identifier:find(':'), identifier:len()) else self.license = 'license:'..identifier end
 
 	ExecuteCommand(('add_principal identifier.%s group.%s'):format(self.license, self.group))
+	
+	Player(self.source).state:set("identifier", self.identifier, true)
+	Player(self.source).state:set("license", self.license, true)
+	Player(self.source).state:set("job", self.job, true)
+	Player(self.source).state:set("group", self.group, true)
+	Player(self.source).state:set("name", self.name, true)
```

Comment:
Why do you keep calling the Player() function?
```suggestion
	local stateBag = Player(self.source).state
	stateBag:set("identifier", self.identifier, true)
	stateBag:set("license", self.license, true)
	stateBag:set("job", self.job, true)
	stateBag:set("group", self.group, true)
	stateBag:set("name", self.name, true)
```

Maybe we can make it so that xPlayer.set is rewritten and uses statebag

Or new function xPlayer.setStatebag/xPlayer.setState

Reply:
I think we could add more states like "handcuffed", "dragged", "onduty" and more

Reply:
> I think we could add more states like "handcuffed", "dragged", "onduty" and more

🫣

Reply:
> Or new function xPlayer.setStatebag/xPlayer.setState
then there would be no point in statebags

Reply:
it would also be a statebag, only simpler. And it would be available from xPlayer

Reply:
But it's just an idea.